src/main/resources/synthea.properties

```diff
@@ -111,7 +111,6 @@ generate.providers.hospice.default_file = providers/hospice.csv
 generate.providers.dialysis.default_file = providers/dialysis.csv
 generate.providers.homehealth.default_file = providers/home_health_agencies.csv
 generate.providers.veterans.default_file = providers/va_facilities.csv
-generate.providers.urgentcare.default_file = providers/urgent_care_facilities.csv
```

Comment:
You need to add back this `generate.providers.urgentcare.default_file` property, but all the other properties should stay as the defaults.